projects/storefrontlib/src/cms-components/myaccount/my-coupons/coupon-claim/coupon-claim.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import {
+  RoutingService,
+  CustomerCouponService,
+  GlobalMessageService,
+  GlobalMessageType,
+} from '@spartacus/core';
+import { Subscription } from 'rxjs';
+
+@Component({
+  templateUrl: './coupon-claim.component.html',
+  selector: 'cx-coupon-claim',
+})
+export class CouponClaimComponent implements OnInit, OnDestroy {
+  subscription: Subscription;
+
+  constructor(
+    private couponService: CustomerCouponService,
```

Comment:
please change all deps to `protected` here

Reply:
updated.